roles/internal/apix/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+apix_feature_repo: mvn:org.fcrepo.apix/fcrepo-api-x-karaf/LATEST/xml/features
+apix_feature:
+  - fcrepo-api-x
+  - fcrepo-indexing-triplestore
+  - fcrepo-service-camel
+
+apix_karaf_etc_dir: "{{ karaf_install_symlink }}/etc"
+
+apix_config:
+  - pid: org.fcrepo.apix.registry.http
+    settings:
+      timeout.socket.ms: 1000
+  - pid: org.fcrepo.camel.indexing.triplestore.cfg
```

Comment:
So for some reason, this is creating a second config file with an extra `.cfg` at the end: `org.fcrepo.camel.indexing.triplestore.cfg.cfg`.  If we can sort this out, we're good to go.  All other functionality is there.  The proxy is up and Fedora + 3store sync'ing is happening from Drupal, just this last bit is preventing the 3store indexer from Fedora..